internal/tester.go

```diff
@@ -152,8 +150,10 @@ func (t *tester) prepareConfig() (*config.TestCase, []config.Resource, error) {
 				if disableImport == "true" {
 					tc.SkipImport = true
 				}
+				if updateParameter == "" {
```

Comment:
Does an `uptest.upbound.io/update-parameter` annotation (or an `uptest.upbound.io/disable-import` annotation) on a dependency resource of an example manifest has any effect?

Reply:
The update is for only root resources now. But import for every resource in the manifest. So, the disable-import has an effect on the dependency resources.